JS9/JS-9-1.js

```diff
@@ -0,0 +1,130 @@
+function newTask() {
+  console.log("========================New Task===========================");
+}
+//1. Создайте класс Employee с полями:
+//firstName — имя сотрудника (строка).
+//lastName — фамилия сотрудника (строка).
+//profession — профессия сотрудника (строка).
+//Приватное поле salary — зарплата сотрудника (число).
+class Employee {
+  constructor(firstName, lastName, profession, salary) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.profession = profession;
+    this._salary = salary;
+  }
+  //2. Реализуйте геттеры и сеттеры для всех полей, включая приватное поле salary.
+  get firstName() {
+    return this._firstName;
+  }
+
+  set firstName(firstName) {
+    this._firstName = firstName;
+  }
+
+  get lastName() {
+    return this._lastName;
+  }
+
+  set lastName(lastName) {
+    this._lastName = lastName;
+  }
+
+  get profession() {
+    return this._profession;
+  }
+
+  set profession(profession) {
+    this._profession = profession;
+  }
+
+  get salary() {
+    return this._salary;
+  }
+
+  set salary(salary) {
+    this._salary = salary;
+  }
+
+  //3. Реализуйте метод getFullName() — возвращающий полное имя сотрудника.
+  getFullName() {
+    return `${this.firstName}, ${this.lastName}`;
+  }
+}
+
+//5. Создайте класс Company с полями:
+//title — название компании (строка).
+//phone — телефон компании (число).
+//address — адрес компании (строка).
+//Приватное поле employees — массив сотрудников (пустой массив на старте).
+class Company {
+  constructor(title, phone, address) {
+    this.title = title;
+    this.phone = phone;
+    this.address = address;
+    this._employees = [];
```

Comment:
И так же далее в классе
```suggestion
    this.#employees = [];
```